### ISSUE TITLE
Links are out of date

### ISSUE THREAD

**Issue by adrivg10** [zju3dv/OnePose_Plus_Plus#53]
Thank you for your amazing work.
It seems that the dataset's [link](https://zjueducn-my.sharepoint.com/:f:/g/personal/zihaowang_zju_edu_cn/ElfzHE0sTXxNndx6uDLWlbYB-2zWuLfjNr56WxF11_DwSg?e=GKI0Df) and [example demo data](https://zjueducn-my.sharepoint.com/:f:/g/personal/12121064_zju_edu_cn/EqgGWk0nHxxAmdv5HLRWTQsBMTLw32Zwr04a2j7NQbI_HQ?e=C0bbpw) are expired (I also cheched the other repository)
Could you renew them? 
Thanks in advance!

**Comment by BWDforce** [comment]
> Thank you for your amazing work. It seems that the dataset's [link](https://zjueducn-my.sharepoint.com/:f:/g/personal/zihaowang_zju_edu_cn/ElfzHE0sTXxNndx6uDLWlbYB-2zWuLfjNr56WxF11_DwSg?e=GKI0Df) and [example demo data](https://zjueducn-my.sharepoint.com/:f:/g/personal/12121064_zju_edu_cn/EqgGWk0nHxxAmdv5HLRWTQsBMTLw32Zwr04a2j7NQbI_HQ?e=C0bbpw) are expired (I also cheched the other repository) Could you renew them? Thanks in advance!

this link has been updated recently

**Comment by adrivg10** [comment]
yes!, the example demo data has already been updated but the dataset's link is still expired 😅

**Comment by carlottaSchiavo** [comment]
Hi ! I would like to test your code but links are still expired. Could you renew them again, please? 
Thanks in advance!
